### PR TITLE
Add qtpyinheritance Python library

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by klauer** [comment]
@conda-forge/help-python, this is ready when you're ready. 👍

File: recipes/qtpyinheritance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - qtpy
+
+test:
+  imports:
+    - qtpyinheritance
+  requires:
+    - pyqt
```
Comment (chrisburr) [inline recipes/qtpyinheritance/meta.yaml:28]: This isn't needed as it's already in the `run` depnendencies:
```suggestion
```

File: recipes/qtpyinheritance/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - qt
```
Comment (chrisburr) [inline recipes/qtpyinheritance/meta.yaml:29]: I suspect this isn't needed as you don't use it directly. It should be included as it is a dependencies of `pyqt`

Comment (klauer) [reply recipes/qtpyinheritance/meta.yaml]: Addressed in 103967b

File: recipes/qtpyinheritance/meta.yaml (hunk 1 above)
Comment (klauer) [reply recipes/qtpyinheritance/meta.yaml:28]: `qtpy` is in the run dependencies, which generically supports either pyqt or pyside, but will fail on import without any bindings installed. That means to test the import we'd need pyqt, for example.

```
$ python -c 'import qtpy'
...

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/klauer/mc/envs/testqt/lib/python3.7/site-packages/qtpy/__init__.py", line 210, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```

Comment (chrisburr) [reply recipes/qtpyinheritance/meta.yaml:28]: As `pyqt` is in `run` this line does nothing

Comment (klauer) [reply recipes/qtpyinheritance/meta.yaml:28]: `qtpy` is in run
`pyqt` is in test

Am I misunderstanding something... ?

Comment (chrisburr) [reply recipes/qtpyinheritance/meta.yaml:28]: No, sorry I just can't read 😆

Comment (klauer) [reply recipes/qtpyinheritance/meta.yaml:28]: No problem - they're easy to confuse!

**Comment by klauer** [comment]
Thanks @chrisburr!

**Pull request merged**